src/game_interpreter.cpp

```diff
@@ -2023,14 +2023,10 @@ bool Game_Interpreter::CommandShakeScreen(RPG::EventCommand const& com) { // cod
 	int speed = com.parameters[1];
 	int tenths = com.parameters[2];
 	bool wait = com.parameters[3] != 0;
+	int shake_cmd = com.parameters.size() > 4 ?
```

Comment:
I would like a comment here referring to 2k.

Reply:
Good idea, added.